pandas/tests/reshape/merge/test_merge.py

```diff
@@ -2761,3 +2763,18 @@ def test_merge_ea_and_non_ea(any_numeric_ea_dtype, join_type):
         }
     )
     tm.assert_frame_equal(result, expected)
+
+
+@pytest.mark.skipif(pa_version_under7p0, reason="need pyarrow")
```

Comment:
IMO I generally prefer using `pytest.importorskip("pyarrow")` so we don't need to change these with each version bump

Reply:
done